plugins/product-reviews/src/admin/components/molecules/ProductReviewDetailsDrawer.tsx

```
@@ -0,0 +1,110 @@
+  );
+
+  const ReviewContent = () => (
+    <div className="flex flex-col gap-2">
+      <Text className="whitespace-pre-wrap">{review.content}</Text>
+    </div>
+  );
+
+  const ImagesValue = () => (
+    <div className="grid grid-cols-3 gap-2">
+      {review.images.map((image, index) => (
+        <img src={image.url} alt={`Review image ${index + 1}`} className="w-full h-full object-cover" />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)